### ISSUE TITLE
[Suggestion] Baritone-style elytra bot

### ISSUE THREAD

**Issue by 0jrv** [MeteorDevelopment/meteor-client#1296]
add baritone elytra fly? (or rocket fly)

https://www.youtube.com/watch?v=34L68ao-b-M

it would be cool if it was added

**Comment by mathmakgakpak** [comment]
sources for elytra baritone fly
https://github.com/bebeli555/AutoBot
https://github.com/bebeli555/CookieClient/tree/main/src/main/java/me/bebeli555/cookieclient/mods/bots/elytrabot

**Comment by mathmakgakpak** [comment]
in my opinion its dumb unless someone will make path finding using fireworks

**Comment by aquakenzie** [comment]
elytra pathfinding was merged into official baritone 2 days ago
